### DISCUSSION TITLE
npx tailwindcss init not working in windows

### DISCUSSION THREAD

**Discussion by theosama717** [tailwindlabs/tailwindcss#14246] [Help] [answered]
when I run npx tailwind css command in the terminal it shows 

npm error code ENOENT
npm error syscall lstat
npm error path C:\Users\cv\AppData\Roaming\npm
npm error errno -4058
npm error enoent ENOENT: no such file or directory, lstat 'C:\Users\cv\AppData\Roaming\npm'
npm error enoent This is related to npm not being able to find a file.
npm error enoent
npm error A complete log of this run can be found in: C:\Users\cv\AppData\Local\npm-cache\_logs\2024-08-24T08_27_09_429Z-debug-0.log

**Comment by wongjn** [comment]
Does this only happen for Tailwind? What it you try to `npx` a different package? The error suggests it is something to do with your npm installation rather than Tailwind specifically.

**Comment by Himanshu-Sorathiya** [comment]
where did you installed your tailwindcss package, i mean globally or just for a workbench/project, if globally then no need to use npx, just use *tailwindcss init* and it will work

**Comment by theosama717** [comment, marked as answer]
Actually i just created an empty folder name 'npm' in C:\Users\cv\AppData\Roaming and npx tailwindcss init started working

**Reply by arshlaanhaq** [reply]
I am still facing error

**Reply by wongjn** [reply]
The `init` command no longer exists in v4. Consider checking [the installation documentation](https://tailwindcss.com/docs/installation/using-vite) that is most relevant to your project to integrate Tailwind v4.

**Reply by arshlaanhaq** [reply]
Ok got it.

**Comment by FurquanAnwer** [comment]
Create tailwind config file manually

**Comment by skido600** [comment]
my is that the tailwind intellisence dont work

**Comment by abderrahimlaribi** [comment]
I am facing this errror, npx tailwindcss init -p
PS C:\Users\DELL\Desktop\CloneGemHunters> npx tailwindcss init -p
npm error could not determine executable to run
npm error A complete log of this run can be found in: C:\Users\DELL\AppData\Local\npm-c
ache\_logs\2025-01-24T16_26_01_565Z-debug-0.log                                        

The dependecies are installed successfully using this command npm install -D tailwindcss postcss autoprefixer, but I can't find the executable  file in .bin

**Reply by arshlaanhaq** [reply]
Just install tailwindcss 4.0 it will not create this error and much easier to install

**Reply by nidinbose** [reply]
same error i have facing how to fix it

**Reply by nidinbose** [reply]
always 4.0 creating error   tailwind config is missing command init also

**Reply by arshlaanhaq** [reply]
if your using vite react you don't have to do npx tailwind init and does not have to create tailwind config file
https://tailwindcss.com/docs/installation/using-vite follow this link to install tailwindcss

**Comment by nidinbose** [comment]
Then how i add extra font styles

**Reply by arshlaanhaq** [reply]
Briefly read the documentation you will understand it. If not, feel free to ask me.

**Comment by nidinbose** [comment]
I really dont know how to add new font file in new update without a config
file

**Reply by arshlaanhaq** [reply]
Use @font 
https://tailwindcss.com/docs/font-family
Here is the link
